pyproject.toml

```diff
@@ -620,3 +621,7 @@ required-imports = ["from __future__ import annotations"]
 
 [tool.ruff.pydocstyle]
 convention = "google"
+
+[tool.ruff.flake8-tidy-imports.banned-api]
+"meltano.cloud.cli.base.cloud.command".msg = "Use `click.command` and `cloud.add_command` instead"
+"meltano.cloud.cli.base.cloud.group".msg = "Use `click.group` and `cloud.add_command` instead"
```

Comment:
Very cool. I like how this serves both to enforce the convention by blocking the import, and to explain what should be done instead.